examples/slide_deck_examples.py

```diff
@@ -0,0 +1,95 @@
+# Databricks notebook source
+# MAGIC %pip install dbl-discoverx
```

Comment:
This line is commented in some notebooks not in others. We should keep in consistent

Reply:
Yes, it should be uncommented always. Thanks!